src/modules/account/components/ManageAccounts/ManageAccounts.js

```diff
@@ -68,4 +72,9 @@ const ManageAccounts = ({ onSelectAccount, onAddAccount, onRemoveAccount }) => {
   );
 };
 
+ManageAccounts.defaultProps = {
+  onSelectAccount: () => null,
```

Comment:
Why `null`? what is the situation where it makes sense to not pass the `onSelectAccount` function, and let it call a pointless function that literally don't nothing?

Reply:
that point less function is avoid the break application and its the correct way to do

Reply:
Please remove the default prop and make the property required.